Hey @laziob I see lint errors:
```
Error: File src/url-params.ts hasn't been formatted with prettier
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! analytics-client@1.2.0 lint: `balena-lint --typescript src/ test/ index.ts && tsc --noEmit`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the analytics-client@1.2.0 lint script.
```

Can you please fix these (`npm run prettify`) and run the lint locally as well (`npm run lint`) before pushing to this remote branch? 

Let's get this and UI one deployed today.

--- a/src/url-params.ts
+++ b/src/url-params.ts
@@ -29,7 +29,7 @@ export class AnalyticsUrlParams {
 	) {
 		const list = inputIdString ? inputIdString.split(deviceIdSeparator) : [];
 		if (list.length > 0) {
-			this.passedDeviceId = list[0]
+			this.passedDeviceId = list[0];
 		}
 
 		if (currentDeviceId) {

